source/Octostache/Templates/BuiltInFunctions.cs

```diff
@@ -30,7 +30,15 @@ static class BuiltInFunctions
             {"substring", TextSubstringFunction.Substring},
             {"truncate", TextManipulationFunction.Truncate},
             {"trim", TextManipulationFunction.Trim},
-            {"uripart", TextManipulationFunction.UriPart}
+            {"uripart", TextManipulationFunction.UriPart},
+            {"versionmajor", VersionParseFunction.VersionMajor},
+            {"versionminor", VersionParseFunction.VersionMinor},
+            {"versionpatch", VersionParseFunction.VersionPatch},
+            {"versionrelease", VersionParseFunction.VersionRelease},
+            {"versionreleaseprefix", VersionParseFunction.VersionReleasePrefix},
```

Comment:
I'm not sure about the naming of the `release` and `releaseprefix`
SemVer refers to this component as `pre-release`.  I worry `release` will be especially confusing in an Octopus context.

Reply:
Good point - I'll change those filter names.